x-pack/plugins/index_lifecycle_management/public/extend_index_management/components/remove_lifecycle_confirm_modal.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiOverlayMask,
+  EuiConfirmModal,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { toastNotifications } from 'ui/notify';
+import { removeLifecycleForIndex } from '../../services/api';
+import { showApiError } from '../../services/api_errors';
+export class RemoveLifecyclePolicyConfirmModal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      policies: [],
+      selectedPolicyName: null,
+      selectedAlias: null,
+    };
+  }
+  oneIndexSelected = () => {
+    return this.props.indexNames.length === 1;
+  };
+  getEntity = oneIndexSelected => {
+    return oneIndexSelected ? (
+      <FormattedMessage
+        id="xpack.indexLifecycleMgmt.indexManagementTable.addLifecyclePolicyConfirmModal.indexMessage"
+        defaultMessage="index"
+      />
+    ) : (
+      <FormattedMessage
+        id="xpack.indexLifecycleMgmt.indexManagementTable.addLifecyclePolicyConfirmModal.indicesMessage"
+        defaultMessage="indices"
+      />
+    );
+  };
+  removePolicy = async () => {
+    const { indexNames, httpClient, closeModal, reloadIndices } = this.props;
+    const target = this. getTarget();
+    try {
+      await removeLifecycleForIndex(indexNames, httpClient);
+      closeModal();
+      toastNotifications.addSuccess(
+        i18n.translate(
+          'xxpack.indexLifecycleMgmt.indexManagementTable.removeLifecyclePolicyConfirmModal.removePolicySuccess',
+          {
+            defaultMessage: 'Removed lifecycle policy from {target}',
+            values: {
+              target
+            },
+          }
+        )
+      );
+      reloadIndices();
+    } catch (err) {
+      showApiError(
+        err,
+        i18n.translate(
+          'xpack.indexLifecycleMgmt.indexManagementTable.removeLifecyclePolicyConfirmModal.removePolicyToIndexError',
+          {
+            defaultMessage: 'Error removing policy',
+          }
+        )
+      );
+    }
+  };
+  getTarget() {
+    const { indexNames } = this.props;
+    const oneIndexSelected = this.oneIndexSelected();
+    const entity = this.getEntity(oneIndexSelected);
+    return oneIndexSelected ? (indexNames[0]) : `${indexNames.length} ${entity}`;
+  }
+  render() {
+
+    const { closeModal } = this.props;
+    const target = this. getTarget();
+    return (
+      <EuiOverlayMask>
+        <EuiConfirmModal
+          title={
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.indexManagementTable.removeLifecyclePolicyConfirmModal.modalTitle"
+              defaultMessage="Remove lifecycle policy from {target}"
```

Comment:
```suggestion
              defaultMessage="Remove lifecycle policy from '{target}'?"
```

it would be nice to display the policy name as well, like:
`Remove lifecycle policy '{policyName}' from '{target}'?`